organisaatio-ui/src/types/react-table-config.d.ts

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
😲
what is this file. why it is needed? where is it taken from? is it modified somehow? 
...dunno details BUT:
might make sense to add it as-is in separate commit with adequate comment. only after that make necessary modification in separate commit.
If there are no changes, do we need to have it in our own repository?

Reply:
I think is needed for React Table. Added as instucted here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-table/Readme.md. I could not find another way to make this work with React Table.